### DISCUSSION TITLE
GREAT combo

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#17840] [Ideas] [answered]
the GREAT combo works like a regular combo counter except that the combo breaks when you hit a judgement worse than GREAT

the GREAT combo by its design factors in accurately hitting the notes, which the regular combo fails to do well

you could use the GREAT combo to note sections where you don't hit things so well but things that a regular combo counter won't catch

**Comment by smoogipoo** [comment, marked as answer]
This sounds completely arbitrary to me. Going to say no to this.
